index.html

```diff
@@ -86,19 +86,36 @@ <h3>Let's see what the Altventurers have made for you</h3>
       </section>
 
       <footer>
-        <a href="https://github.com/elysiumdelivery/altare-tcg"
-          ><svg
-            role="img"
-            viewBox="0 0 24 24"
-            xmlns="http://www.w3.org/2000/svg"
+        <div id="footer-nav">
+          <a id="footer-download-pdf" class="download-button" href="#">
```

Comment:
Are the buttons on the fancy menu super big on purpose? If so, could the link text be aligned at the bottom for both instead of just one?

Reply:
No, something broke here - sorry. Nice catch. I will fix.

Reply:
Updated on the homepage and made it fit in better with the existing splash page setup and animation.